src/batch/src/task/task_execution.rs

```diff
@@ -299,7 +302,7 @@ pub struct BatchTaskExecution<C> {
     failure: Arc<Mutex<Option<RwError>>>,
 
     /// Shutdown signal sender.
-    shutdown_tx: Mutex<Option<Sender<u64>>>,
+    shutdown_tx: Mutex<Option<Sender<String>>>,
```

Comment:
May wrap the message in an `Err` to make the code more readable.

Reply:
The query result sender requires a `String` type here, so I'll keep this as is and add some comments about it.